### PR TITLE
CHAD-4257 Release EcoSmart 4-button remote DTH

### PR DESCRIPTION
**Pull request by BroderickCarlin**

### CODE REVIEW

**Review by greens — DISMISSED** [review]

File: devicetypes/smartthings/ecosmart-4button-remote.src/ecosmart-4button-remote.groovy [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+		// remember to ignore the next button 3 message we get
+		state.ignoreNextButton3 = true
+		buttonNumber = 4
+	}
+
+
+	if (buttonNumber != 0) {
+		// Create old style
+		def descriptionText = "${getButtonName(buttonNumber)} was pushed"
+		//result = [name: "button", value: "pushed", data: [buttonNumber: buttonNumber], descriptionText: descriptionText, isStateChange: true]
```
Comment (dkirker) [inline devicetypes/smartthings/ecosmart-4button-remote.src/ecosmart-4button-remote.groovy]: Any reason for dropping the aggregate event?

File: devicetypes/smartthings/ecosmart-4button-remote.src/ecosmart-4button-remote.groovy [hunk 2]
```diff
@@ -0,0 +1,194 @@
+import groovy.json.JsonOutput
```
Comment (dkirker) [inline devicetypes/smartthings/ecosmart-4button-remote.src/ecosmart-4button-remote.groovy:17]: Don't forget the ST copyright and license block!

**Review by dkirker — COMMENTED** [review]
Took a quick look and left a few comments in-line!

**Comment by BroderickCarlin** [comment]
@greens and @dkirker, can I get one final 👀

**Review by greens — APPROVED** [review]

**Pull request merged**